content/influxdb/v2.0/get-started.md

```diff
@@ -235,6 +239,8 @@ If you rename the binaries, all references to `influx` and `influxd` in this doc
      Active: active (running)
    ```
 
+When installed as a service, InfluxDB stores data in `/var/lib/influxdb/`.
```

Comment:
Is it just data or all InfluxDB files/directories (`configs/`, `engine/`, `influxd.bolt`)?

Reply:
Looks like configs are in `$HOME/.influxdbv2/` when installed on Ubuntu using `.deb`.

Reply:
`engine/` and `influx.bolt` are both in `/var/lib/influxdb`.

Reply:
Editing the PR now...